adserver/templates/adserver/includes/widgets/advertisement-form-option.html

```diff
@@ -0,0 +1,50 @@
+{# Adapted from bootstrap4/layout/checkboxselectmultiple.html in crispy forms #}
```

Comment:
Not 100% sure how to review this. Might be useful to note what was changed here (eg. `Changes on lines 6-9`)

Reply:
There are A LOT of changes. This file is now specific to advertisements.